pandas/io/json/normalize.py

```diff
@@ -149,6 +175,20 @@ def json_normalize(data, record_path=None, meta=None,
     1  NaN         NaN      Regner        NaN       Mose       NaN
     2  2.0  Faye Raker         NaN        NaN        NaN       NaN
 
+    >>> from pandas.io.json import json_normalize
+    >>> data = [{'id': 1,
+    ...          'name': {'first': 'Coleen', 'last': 'Volk'},
+    ...          "fitness": {"height":130, "weight":60}},
+    ...         {'name': {'given': 'Mose', 'family': 'Regner'},
+    ...          "fitness": {"height":130, "weight":60}},
+    ...         {'id': 2, 'name': 'Faye Raker',
+    ...          "fitness": {"height":130, "weight":60}}]
+    >>> json_normalize(data, max_level=1, ignore_keys=["name"])
```

Comment:
After giving this some more thought...can we split this PR up? I think this one should just focus on `max_level` and a subsequent PR can do `ignore_keys`. Will make review and communication out easier